test/FastTests/Server/Basic/DatabaseRecordDebugPackageInclusionTest.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Raven.Client.ServerWide;
+using Raven.Server.Documents.Handlers.Debugging;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.Server.Basic
+{
+    public class DatabaseRecordDebugPackageInclusionTest : RavenTestBase
+    {
+        public DatabaseRecordDebugPackageInclusionTest(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void EnsureIncludedInDatabaseRecordDebugPackage()
+        {
+            // Only add to this list after deciding whether the new property should be exposed to the debug package
+            // If it should be added to the debug package, add it to DatabaseRecordHandler.FieldsThatShouldBeExposedForDebug
+            var decidedFields = new HashSet<string>() {
```

Comment:
use the `FieldsThatShouldBeExposedForDebug` and  concat with "unsafe" fields

Reply:
done